docs/YAML_documents/yaml_structure.md

```diff
@@ -0,0 +1,48 @@
+YAML parser will receive a test yaml file as an input. 
+
+A test yaml file should have the following parameters:
+
+* test_id
+* object_size
+* part_size
+* range_read
+* part_range
+* min_runtime
+* sessions_per_node
+* sessions
+* tool
+
+---
+
+TEST_ID can be any unique string which can be treated as filtering through execution logs.  
```

Comment:
its mentioned as lower case test_id above and here upper case, so which is to be used?

Reply:
Changed to upper case